key_update test fail after integrate #166

@vinmaciel, would you please take a look and let us know if it is expected or not?

REF: https://github.com/DMTF/libspdm/pull/166

[  ERROR   ] --- 0x8000000000000007 != 0x8000000000000010
[   LINE   ] --- C:\home\DMTF\libspdm\unit_test\test_spdm_requester\key_update.c:4143: error: Failure!
[  ERROR   ] --- 0x8000000000000007 != 0x0
[   LINE   ] --- C:\home\DMTF\libspdm\unit_test\test_spdm_requester\key_update.c:4216: error: Failure!
[  ERROR   ] --- 0x3 != 0x0
[   LINE   ] --- C:\home\DMTF\libspdm\unit_test\test_spdm_requester\key_update.c:4279: error: Failure!
[  ERROR   ] --- 0x8000000000000007 != 0x0
[   LINE   ] --- C:\home\DMTF\libspdm\unit_test\test_spdm_requester\key_update.c:4405: error: Failure!
[  PASSED  ] 30 test(s).
[  FAILED  ] spdm_requester_key_update_tests: 4 test(s), listed below:
[  FAILED  ] test_spdm_requester_key_update_case29
[  FAILED  ] test_spdm_requester_key_update_case30
[  FAILED  ] test_spdm_requester_key_update_case31
[  FAILED  ] test_spdm_requester_key_update_case33

--- a/unit_test/test_spdm_requester/key_update.c
+++ b/unit_test/test_spdm_requester/key_update.c
@@ -4566,15 +4566,15 @@ int spdm_requester_key_update_test_main(void)
 		// Error response: SPDM_ERROR_CODE_INVALID_REQUEST
 		cmocka_unit_test(test_spdm_requester_key_update_case28),
 		// Always SPDM_ERROR_CODE_BUSY
-		cmocka_unit_test(test_spdm_requester_key_update_case29),
+		//cmocka_unit_test(test_spdm_requester_key_update_case29),
 		// SPDM_ERROR_CODE_BUSY + Successful response
-		cmocka_unit_test(test_spdm_requester_key_update_case30),
+		//cmocka_unit_test(test_spdm_requester_key_update_case30),
 		// Error response: SPDM_ERROR_CODE_REQUEST_RESYNCH
-		cmocka_unit_test(test_spdm_requester_key_update_case31),
+		//cmocka_unit_test(test_spdm_requester_key_update_case31),
 		// Always SPDM_ERROR_CODE_RESPONSE_NOT_READY
 		cmocka_unit_test(test_spdm_requester_key_update_case32),
 		// SPDM_ERROR_CODE_RESPONSE_NOT_READY + Successful response
-		cmocka_unit_test(test_spdm_requester_key_update_case33),
+		//cmocka_unit_test(test_spdm_requester_key_update_case33),
 		// Unexpected errors
 		cmocka_unit_test(test_spdm_requester_key_update_case34),
 	};

